organizations_test.go

```diff
@@ -0,0 +1,48 @@
+package awx
+
+import (
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
```

Comment:
Introduce a new test dependency should think twice. I'm ok to this, but I think it should be another single PR for this change.

Reply:
So I figured it out, the organization_test.go is special, this is an integration test powered by a test suite to make code DRY. However, its not enabled by default unless you specify the integration tag. See the latest commits.

Do you think its an idea to make a mock test instead?

I have a working AWX server, deployed with script from https://github.com/davidfischer-ch/awx-go/commit/d299f2aad22d6b6cfad21568f84853a567735a67.

Reply:
OK?

Reply:
Yes, for the sake of "safe by default", if it's been disabled by default, and it also works if enabled, I'll be OK.

Reply:
Indeed, but since its an integration test it needs to be working properly an actual AWX server, up and responding to http://127.0.0.1:80. Hence the script to provision it, using docker-compose.